server/xdg_wm_base.h

```diff
@@ -0,0 +1,10 @@
+#ifndef ZWAYLAND_XDG_WM_BASE
```

Comment:
```suggestion
#ifndef ZWAYLAND_XDG_WM_BASE_H
```
ファイル内の ZWAYLAND_XDG_WM_BASE も同様

Reply:
Thanks